Parsers/ASimUserManagement/Parsers/ASimUserManagementSentinelOne.yaml

```diff
@@ -0,0 +1,134 @@
+EquivalentBuiltInParser: _Im_UserManagement_SentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventTypeLookup = datatable (
+      activityType_d: real,
+      EventType: string,
+      EventOriginalType: string,
+      EventSubType: string
+  )[
+      23, "UserCreated", "User Added", "",
+      24, "UserModified", "User Modified", "MultipleProperties",
+      25, "UserDeleted", "User Deleted", "",
+      37, "UserModified", "User modified", "UserModified",
+      102, "UserDeleted", "User Deleted", "",
+      110, "UserModified", "Enable API Token Generation", "NewPermissions",
+      111, "UserModified", "Disable API Token Generation", "PreviousPermissions",
+      140, "UserCreated", "Service User creation", "",
+      141, "UserModified", "Service User modification", "MultipleProperties",
+      142, "UserDeleted", "Service User deletion", "",
+      3522, "GroupCreated", "Ranger Deploy - Credential Group Created", "",
+      3523, "GroupModified", "Ranger Deploy -Credential Group Edited", "MultipleProperties",
+      3524, "GroupDeleted", "Ranger Deploy - Credential Group Deleted", "",
+      3710, "PasswordReset", "User Reset Password with Forgot Password from the Login", "",
+      3711, "PasswordChanged", "User Changed Their Password", "",
+      3715, "PasswordReset", "User Reset Password by Admin Request", "",
+      5006, "GroupDeleted", "Group Deleted", "",
+      5008, "GroupCreated", "User created a Manual or Pinned Group", "",
+      5011, "GroupModified", "Group Policy Reverted", "Newpolicy",
+  ];
+  let parser = (disabled: bool=false) {
+      SentinelOne_CL
+      | where event_name_s == "Activities."
+          and activityType_d in (23, 24, 25, 37, 102, 110, 111, 140, 141, 142, 3522, 3523, 3524, 3710, 3711, 3715, 5006, 5008, 5011)
+      | parse-kv DataFields_s as (byUser: string, username: string, email: string, ipAddress: string, group: string, groupName: string, name: string, oldDescription: string, oldRole: string, description: string, role: string, userScope: string, scopeLevelName: string, scopeName: string, roleName: string, modifiedFields: string, deactivationPeriodInDays: string, descriptionChanged: string, groupType: string) with (pair_delimiter=",", kv_delimiter=":", quote='"')
+      | parse description_s with * "with id=" id: string "," restOfMessage
+      | lookup EventTypeLookup on activityType_d
+      | extend 
+          ActorUsername = iff(activityType_d == 102, "SentinelOne", coalesce(byUser, username, email)),        
+          GroupName = coalesce(group, groupName, name),
+          TargetUsername = iff(isnotempty(byUser), username, ""),
+          PreviousPropertyValue = coalesce(oldDescription, oldRole),
+          NewPropertyValue = coalesce(description, role)
+      | extend GroupName = iff(GroupName == "null", "", GroupName)
+      | project-rename
+          EventStartTime = createdAt_t,
+          DvcIpAddr = ipAddress,
```

Comment:
https://learn.microsoft.com/en-us/azure/sentinel/normalization-schema-user-management#source-fields
please check if we can use Src fields

Reply:
Done